packages/modules/event-bus-local/src/services/__tests__/event-bus-local.js

```diff
@@ -65,6 +65,124 @@ describe("LocalEventBusService", () => {
           hi: "5678",
         })
       })
+
+      it("should stage an event if data consists of eventGroupId", () => {
+        eventBus = new LocalEventBusService(
+          moduleDeps,
+          {},
+          { resources: "shared" }
+        )
+
+        const stageEventFn = jest.spyOn(eventBus, "stageEvent")
+
+        eventBus.eventEmitter_.emit.mockImplementationOnce((data) => data)
+
+        eventBus.emit("test-event", { test: "1234", eventGroupId: "test" })
+
+        expect(eventBus.eventEmitter_.emit).not.toHaveBeenCalled()
+        expect(stageEventFn).toHaveBeenCalledTimes(1)
+        expect(stageEventFn).toHaveBeenCalledWith("test", "test-event", {
+          test: "1234",
+        })
+
+        eventBus.emit("test-event", { test: "1234", eventGroupId: "test" })
```

Comment:
suggestion: can we also emit without group id to ensure the behaviour is fully tested ?